examples/psyad/lfric/tangent_linear_tweaked/tl_kinetic_energy_gradient_kernel_mod_tweaked.F90

```diff
@@ -201,8 +201,8 @@ subroutine tl_kinetic_energy_gradient_code(nlayers,       &
         end do
 
         ! Calculation
-        mul1(3) = matmul( jac(:,:,qp1,qp2), ls_u_at_quad )
-        mul2(3) = matmul( jac(:,:,qp1,qp2), u_at_quad )
+        mul1 = matmul( jac(:,:,qp1,qp2), ls_u_at_quad )

```

Comment:
Good spot. Presumably we don't compiler these kernels which is how this got through? Probably not something for this PR though.

Reply:
This is what I think yes.